src/rg3d/ui/check_box.md

```diff
@@ -1 +1,67 @@
-# Check box (WIP)
\ No newline at end of file
+# Check box
+
+To create a checkbox you should do something like this:
+
+```rust
+# use rg3d::{
+#     core::pool::Handle,
+#     gui::{check_box::CheckBoxBuilder, widget::WidgetBuilder, UiNode, UserInterface},
+# };
+fn create_checkbox(ui: &mut UserInterface) -> Handle<UiNode> {
+    CheckBoxBuilder::new(WidgetBuilder::new())
+        .checked(false)
+        .build(&mut ui.build_ctx())
+}
+```
+
+## Message handling
+
+Checkboxes are not static widget and have muliple states.
+To handle a message from a checkbox, you need to read a 
+`CheckBoxMessage::checked` variable. To do so, you can do
+something like this:
+
+```rust
+use rg3d::{
+    core::pool::Handle,
+    engine::{framework::GameState, Engine},
+    gui::{
+        check_box::CheckBoxBuilder,
+        message::{CheckboxMessage, UiMessage, UiMessageData},
+        widget::WidgetBuilder,
+        UiNode,
+    },
+};
```

Comment:
Add `# ` at each marked line to hide imports please. Also please add `# extern crate rg3d` before imports to make tests work.